### ISSUE TITLE
Does the package still in active development?

### ISSUE THREAD

**Issue by redeemefy** [pichillilorenzo/JavaScriptEnhancements#78]

**Comment by dalisoft** [comment]
I don't think so, this project last commit more than year and this looks like project not in active development, just in maintaince mode (accepts PR bug-fixes only)

**Comment by romeopeter** [comment]
The last commit on this package was two years ago, it no longer in development anymore. The package works well and has a lot of great features to it. I wonder he/she stopped the development.
